libs/community/langchain_community/utilities/vertexai.py

```diff
@@ -105,3 +106,19 @@ def get_client_info(module: Optional[str] = None) -> "ClientInfo":
         client_library_version=client_library_version,
         user_agent=f"langchain/{client_library_version}",
     )
+
+
+def load_image_from_gcs(path: str, project: Optional[str] = None) -> "Image":
+    """Loads im Image from GCS."""
+    try:
+        from google.cloud import storage
+    except ImportError:
+        raise ImportError("Could not import google-cloud-storage python package.")
+    from vertexai.preview.generative_models import Image
+
+    gcs_client = storage.Client(project=project)
+    pieces = path.split("/")
```

Comment:
It would probably be good to create utility functions that can split and merge GCS paths using Regex (or use one if it already exists).

There are some in Document AI Toolbox for a reference, probably doesn't make sense to import it just for these functions.

https://github.com/googleapis/python-documentai-toolbox/blob/7d30aa5000c0b42da45b9de7edf6ee75362f8e5f/google/cloud/documentai_toolbox/utilities/gcs_utilities.py#L152